lib/Stats.js

```diff
@@ -80,6 +80,17 @@ class Stats {
 			typeof v !== "undefined" ? v :
 			typeof options.all !== "undefined" ? options.all : def;
 
+		const testAgainstGivenOption = (item) => {
+			if(typeof item === "string") {
+				const regExp = new RegExp(`[\\\\/]${item.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&")}([\\\\/]|$|!|\\?)`); // eslint-disable-line no-useless-escape
+				return ident => regExp.test(ident);
+			}
+			if(item && typeof item === "object" && typeof item.test === "function")
+				return ident => item.test(ident);
+			if(typeof item === "function")
```

Comment:
Could you add `true` as possible value?

So `excludeModules: true` / `excludeAssets: true` hides all assets.